package.json

```diff
@@ -178,7 +178,7 @@
     "denymount": "^2.3.0",
     "docker-modem": "^2.0.2",
     "docker-progress": "^4.0.0",
-    "docker-qemu-transpose": "^1.0.1",
+    "docker-qemu-transpose": "^1.0.2-25-invalid-containerPort-bef7d7d683de359536bb334fce603dbae336696d",
```

Comment:
This will be changed to `"^1.0.2"` when a reviewer kindly approves this other PR: :-)
https://github.com/balena-io-modules/docker-qemu-transpose/pull/26